assets/src/stories-editor/components/resizable-box/index.js

```diff
@@ -54,7 +54,7 @@ const EnhancedResizableBox = ( props ) => {
 	} = props;
 
 	const isImage = 'core/image' === blockName;
-	const isBlockWithText = BLOCKS_WITH_TEXT_SETTINGS.includes( blockName );
+	const isBlockWithText = BLOCKS_WITH_TEXT_SETTINGS.includes( blockName ) || 'core/code' === blockName;
```

Comment:
Are the verse and preformatted blocks also relevant here?

Reply:
Not at this moment since these two blocks don't have the resizing anymore (see the first point in the description). If at some point we'll add background color or any other feature that would make resizing have a point then we could add it back.